erts/etc/win32/nsis/Makefile

```diff
@@ -69,6 +69,7 @@ REDIST_DLL_NAME=$(shell (sh ./dll_version_helper.sh -n "$(NICEREDISTFILE)" || ec
 # $(info $$REDIST_FILE = [${REDIST_FILE}])
 
 release_spec:
+	makensis.exe -version; \
```

Comment:
Why this?

Reply:
A double-check to see the version being used. I can remove it.